libs/redux/userTerrains/api.ts

```diff
@@ -0,0 +1,14 @@
+import clientApi from '../clientApi';
+
+const UserTerriansAPI = clientApi.injectEndpoints({
```

Comment:
Problem 1:
Spelling mistake, `UserTerriansAPI` should be `UserTerrainsAPI`

Problem 2:
Naming convention `UserTerrainsAPI` should be `UserTerrainsApi`